src/data/testData.js

```diff
@@ -1,10 +1,479 @@
 ///Test data goes here - use this template to export
 
 
-//change variables used
-// module.exports = {
-//     users,
-//     hydrationData,
-//     activityData,
-//     sleepData
-//   }
\ No newline at end of file
+const travelers = [
```

Comment:
@butlertree You could probably slim down your test data set. You probably only need 3 travelers—one with 1 trip/destination, one with multiple trips/destinations, and one with no trips/destinations. I don't think it's necessarily bad to have more data than that, but it can be harder to keep track of it if there's more than you need, if that makes sense.